near-sdk-sim/src/runtime.rs

```diff
@@ -268,7 +271,7 @@ impl RuntimeStandalone {
             random_seed: Default::default(),
             epoch_id: EpochId::default(),
             current_protocol_version: PROTOCOL_VERSION,
-            config: Arc::from(self.genesis.runtime_config.clone()),
+            config: RuntimeConfig::from_protocol_version(&self.runtime_config, PROTOCOL_VERSION),
```

Comment:
The thing with lazy_static that happens inside `RuntimeConfig::from_protocol_version` feels a bit odd. Seems like tests might start with different genesiss_config, and then config of one test will override config of the other?

Reply:
:thinking: I think that's exactly the reason why https://github.com/near/nearcore/pull/4263 fails

Reply:
Added a fix for that to https://github.com/near/nearcore/pull/4263 (not that we can immediately use that in the SDK)

Reply:
It was mostly for mainnet upgrade. I guess we can also override Default for GenesisConfig to use this instead of applying it in the produce block. This way the user can override the config if needed.